### PR TITLE
Param table component

### PR DESCRIPTION
**Pull request by RutaUrbonaviciute**

### CODE REVIEW

**Review by saulyz — CHANGES_REQUESTED** [review]

File: src/components/animal/ParamTable.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { Box, makeStyles, Typography } from '@material-ui/core';
+
+export default function ParamTable({ details }: ParamTableProps) {
+    const classes = useStyles();
+
+    return (
+        <>
+            {details &&
+                details.map((item, index) => (
+                    <Box key={index} display="flex" justifyContent="space-between" className={classes.root} p={1}>
```
[the remark this answers is not in the document]
Comment (RutaUrbonaviciute) [reply src/components/animal/ParamTable.tsx]: I changed "root" class to 'item" and for the root element i don't think i need any styling, so maybe div tag with root class is not necessary in this case?

File: src/utils/animalDetails.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { Animal } from '../graphql/types';
```
Comment (saulyz) [inline src/utils/animalDetails.ts:1]: Let's move this code to `animals.ts`. So there will be 1 source of utilities nicely grouped in 1 file.

File: src/utils/paramTable.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Event } from '../graphql/types';
```
Comment (saulyz) [inline src/utils/paramTable.ts:1]: Let's move this code to `events.ts`. So there will be 1 source of utilities nicely grouped in 1 file.
In the future more logic can be added to render various types of event.

**Review by saulyz — CHANGES_REQUESTED** [review]
waiting for the changes

**Review by saulyz — APPROVED** [review]

**Comment by saulyz** [comment]
Please update your forked branch with new `main` and then push.
Your codebase needs to be updated with the solved lint problems.

**Pull request merged**